Readme.md

```diff
@@ -304,12 +305,61 @@ Gather up the message that was sent with the http request, and save it in
 `req.body`.  This call is safe to call more than once, but sets body only the
 first time.
 
+### restiq.mw.discardBody( req, res, next )
+
+Reads and discards request body forcing the end event on the request.
```

Comment:
I skimmed this so I missed the back tick request. That ended up in a  separate commit 4928e3801cd304e85d86d0081f3ca610d426c738